Documentation/ceph-upgrade.md

```diff
@@ -270,6 +270,15 @@ below, which you should change to match where your images are located.
 You can also remove the `ROOK_CSI_CEPHFS_IMAGE` and `ROOK_CSI_RBD_IMAGE` `env` variables that are
 no longer used in Rook.
 
+if you have configured the kubelet to use other than `/var/lib/kubelet` please
```

Comment:
nit: make it a complete sentence by capitalizing "If" and end with a period.

Reply:
updated